python/cugraph/dask/core.py

```diff
@@ -0,0 +1,184 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import logging
+import numba.cuda
+import time
+import os
+from threading import Lock, Thread
+import cugraph
```

Comment:
Do we have any guidelines on import orders? In this file cugraph is imported last and in the pagerank.py below, it's imported first. Not a critical thing for this PR, but enforcing some consistency will help code readability.

Reply:
This order is the proper one. 

According to PEP8 :
```
Imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports.
````
https://www.python.org/dev/peps/pep-0008/#imports

Notice that it is pretty close to what the order of C++ header should be ideally.

Reply:
Great, and then, it might be great if we can at least encourage people to follow this order (enforcing may be too much for some people).

Reply:
Agreed :) Let's bring that to the next sync meeting. This would fit into the refactoring document too.

Reply:
google doc has been updated